docs/integrations/builtin/credentials/google/oauth-single-service.md

```diff
@@ -60,6 +69,8 @@ In Google Cloud:
 	1. If using Google Perspective or Google Ads: [Request API Access for Perspective](https://developers.perspectiveapi.com/s/docs-get-started){:targe=_blank .external-link} or a [Developer Token for Ads](https://developers.google.com/google-ads/api/docs/first-call/dev-token){:targe=_blank .external-link}.  
```

Comment:
```suggestion
	1. If using Google Perspective or Google Ads: [Request API Access for Perspective](https://developers.perspectiveapi.com/s/docs-get-started){:target=_blank .external-link} or a [Developer Token for Ads](https://developers.google.com/google-ads/api/docs/first-call/dev-token){:target=_blank .external-link}.  
```

Reply:
This part was copied from https://docs.n8n.io/integrations/builtin/credentials/google/oauth-generic/
So need to adjust the `target` there too! (will make another PR for this)